pixeltable/func/callable_function.py

```diff
@@ -95,7 +95,15 @@ def name(self) -> str:
 
     def overload(self, fn: Callable) -> CallableFunction:
         if self.self_path is None:
-            raise excs.Error('@overload can only be used with module UDFs (not locally defined UDFs)')
+            raise excs.Error('`overload` can only be used with module UDFs (not locally defined UDFs)')
+        if self.is_batched:
+            raise excs.Error('`overload` cannot be used with batched functions')
+        if self.is_method or self.is_property:
+            raise excs.Error('`overload` cannot be used with cannot be used with `is_method` or `is_property`')
```

Comment:
cannot be used with cannot be used with

Reply:
Fixed.
<!-- Reviewable comment -OEWkB7g3Co94Hsczd-l:bt2vr1b -->
<!-- Sent from Reviewable.io -->